### PR TITLE
Tidied up tests.

### PR DESCRIPTION
**Pull request by SidRoberts**
Hello!

* Type: code quality
* Link to issue:

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [x] I wrote some tests for this PR
- [-] I have updated the relevant CHANGELOG
- [-] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Small description of change:

Thanks

### CODE REVIEW

File: tests/integration/Mvc/Model/Behavior/NotifyCest.php [hunk 1]
```diff
@@ -1,33 +0,0 @@
-<?php
```
Comment (niden) [inline tests/integration/Mvc/Model/Behavior/NotifyCest.php]: Why are these deleted?

File: tests/unit/Mvc/Collection/AggregateCest.php [hunk 2]
```diff
@@ -1,31 +0,0 @@
-<?php
```
Comment (niden) [inline tests/unit/Mvc/Collection/AggregateCest.php]: Are these deleted in anticipation of the new mongo collection object?

File: tests/unit/Mvc/Dispatcher/CallActionMethodCest.php [hunk 3]
```diff
@@ -1,31 +0,0 @@
-<?php
```
Comment (niden) [inline tests/unit/Mvc/Dispatcher/CallActionMethodCest.php]: Same why are these deleted?

**Review by niden — CHANGES_REQUESTED** [review]

File: tests/integration/Mvc/Model/Behavior/NotifyCest.php (hunk 1 above)
Comment (SidRoberts) [reply tests/integration/Mvc/Model/Behavior/NotifyCest.php]: It's an abstract class so we can't test it directly and it is covered by the subclasses SoftDelete and Timestampable anyway.

File: tests/unit/Mvc/Collection/AggregateCest.php (hunk 2 above)
Comment (SidRoberts) [reply tests/unit/Mvc/Collection/AggregateCest.php]: All of the cests deleted from unit/Mvc/ also exist in integration/. As they require the DI so they should be in integration/ so these have been deleted.

File: tests/unit/Mvc/Dispatcher/CallActionMethodCest.php (hunk 3 above)
Comment (SidRoberts) [reply tests/unit/Mvc/Dispatcher/CallActionMethodCest.php]: See Collection comment.

**Review by niden — APPROVED** [review]

**Comment by niden** [comment]
Thank you @SidRoberts
